NEWS.md

```diff
@@ -570,7 +570,8 @@ A lot of work has gone into making Ractors more stable, performant, and usable.
 
 * ZJIT
     * Introduce an [experimental method-based JIT compiler](https://docs.ruby-lang.org/en/master/jit/zjit_md.html).
-      To enable ZJIT on supported platforms, supply the `--zjit` option or call `RubyVM::ZJIT.enable` at runtime.
+      Where available, ZJIT can be enabled at runtime with the `--zjit` option or by calling `RubyVM::ZJIT.enable`.
+      When building Ruby from source, Rust 1.85.0 or later is required to include ZJIT support.
```

Comment:
```suggestion
      When building Ruby, Rust 1.85.0 or later is required to include ZJIT support.
```

It applies to not only source builds but also tarball builds. Adding "from source" seems just misleading.